libs/blocks/how-to/how-to.js

```diff
@@ -47,7 +47,19 @@ const setJsonLd = (heading, description, mainImage, stepsLd) => {
 };
 
 const getImage = (el) => el.querySelector('picture') || el.querySelector('a[href$=".svg"');
-const getVideo = (el) => el.querySelector('a[href*=".mp4"]');
+const getVideo = (el) => {
+  const video = el.querySelector('video');
+  if (video) {
+    const src = video.getAttribute('data-video-source');
+    video.appendChild(createTag('source', { src, type: 'video/mp4' }));
+    return video;
+  }
+  const adobeTV = el.querySelector('.milo-video');
+  if (adobeTV) {
+    return adobeTV;
+  }
+  return null;
```

Comment:
Wouldn't it be enough to just change the selector? For example the source child of the video should be added via an intersection observer